db-client/src/main/java/com/splicemachine/db/client/net/NetConnection.java

```diff
@@ -952,16 +956,27 @@ private void flowSecurityCheckAndAccessRdb(int securityMechanism,
     }
 
     private void flowSecurityCheck(int securityMechanism,
-                                               String user,
-                                               String password,
-                                               byte[] encryptedUserid,
-                                               byte[] encryptedPassword) throws SqlException {
+                                   String user,
+                                   String password,
+                                   byte[] encryptedUserid,
+                                   byte[] encryptedPassword) throws SqlException {
+        flowSecurityCheck(securityMechanism, user, password, encryptedUserid, encryptedPassword, null);
+    }
+
+    private void flowSecurityCheck(int securityMechanism,
+                                   String user,
+                                   String password,
+                                   byte[] encryptedUserid,
+                                   byte[] encryptedPassword,
+                                   String pluginName
```

Comment:
i would suggest to rename `pluginName` to `securityPlugin` or `securityPluginName` to make things clear in all code paths that this is about the security plugin.
please also add a documentation of this parameter. What are the allowed parameters for pluginName?

Reply:
done